src/core/algorithms/fd/pfdtane/pfdtane.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+
+#include "algorithms/fd/pli_based_fd_algorithm.h"
+#include "algorithms/fd/tane/lattice_level.h"
+#include "config/error/type.h"
+#include "config/error_measure/type.h"
+#include "config/max_lhs/type.h"
+#include "enums.h"
+#include "model/table/position_list_index.h"
+#include "model/table/relation_data.h"
+
+namespace algos {
+using boost::dynamic_bitset;
```

Comment:
in pfdtane.cpp I have `using Cluster = model::PositionListIndex::Cluster;` just to have a shorthand version of Cluster. Is that ok?

Reply:
Yes, I will write the problem in more detail: when we add such using in a header, it will also be in all headers that include this header. We definitely don't want this. 
You can use such usings in source files.